### ISSUE TITLE
Allow <option> Tag to be Written in a Single Line

### ISSUE THREAD

**Issue by irfanfadilah** [vuejs/eslint-plugin-vue#1740]
**What rule do you want to change?**

`vue/singleline-html-element-content-newline`

**Does this change cause the rule to produce more or fewer warnings?**

`Fewer`

**How will the change be implemented? (New option, new default behavior, etc.)?**

By appending `lib/utils/inline-non-void-elements.json`

**Please provide some example code that this change will affect:**

```vue
<!-- Valid Option Tag -->
<option value="vue">Vue</option>
```

**What does the rule currently do for this code?**

`<option>` tag cannot be written in a single line.

**What will the rule do after it's changed?**

Allow `<option>` tag to be written in a single line.

**Additional context**

In the current rules, writing `<option>` tags are taking unnecessary spaces.

References and examples:

1. https://html.spec.whatwg.org/multipage/form-elements.html
2. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option
3. https://www.w3schools.com/tags/tag_option.asp

**Comment by ota-meshi** [comment]
Thank you for posting this issue.
But, line breaks do not seem to affect the display.

https://sfc.vuejs.org/#eyJBcHAudnVlIjoiPHRlbXBsYXRlPlxuICA8c2VsZWN0PlxuICAgIDxvcHRpb24+QTwvb3B0aW9uPlxuICAgIDxvcHRpb24+XG4gICAgXHRBXG4gICAgPC9vcHRpb24+XG4gIDwvc2VsZWN0PlxuPC90ZW1wbGF0ZT4iLCJpbXBvcnQtbWFwLmpzb24iOiJ7XG4gIFwiaW1wb3J0c1wiOiB7XG4gICAgXCJ2dWVcIjogXCJodHRwczovL3NmYy52dWVqcy5vcmcvdnVlLnJ1bnRpbWUuZXNtLWJyb3dzZXIuanNcIlxuICB9XG59In0=

If you don't want to add line breaks to the `<option>` if you like, you can use the `ignores` option.

https://eslint.vuejs.org/rules/singleline-html-element-content-newline.html#options
